.github/workflows/check-amdllpc-docker.yml

```diff
@@ -49,4 +51,18 @@ jobs:
                             --build-arg LLPC_REPO_NAME="${GITHUB_REPOSITORY}"
                             --build-arg LLPC_REPO_REF="${GITHUB_REF}"
                             --build-arg LLPC_REPO_SHA="${GITHUB_SHA}"
+                            --build-arg FEATURES="${{ matrix.feature-set }}"
                             --tag llpc/ci-shaderdb
+      - name: Copy code coverage report to host and upload to GCS
+        if: contains(matrix.feature-set, '+coverage') && github.event.pull_request.number
+        run: |
+          docker run -v "$HOME:/host" llpc/ci-shaderdb 'bash' '-c' 'cp -r /vulkandriver/coverage_report /host/ && chown -R runner /host/coverage_report'
```

Comment:
nit: can we put the `chown` outside of docker (IE run it in the runner itself)? I think this would be cleaner as docker does not have to learn about the runner username.